app/internal/page/donor/can_evidence_be_uploaded.go

```diff
@@ -0,0 +1,41 @@
+package donor
+
+import (
+	"net/http"
+
+	"github.com/ministryofjustice/opg-go-common/template"
+	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/actor"
+	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/form"
+	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/page"
+	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/validation"
+)
+
+type canEvidenceBeUploadedData struct {
+	App     page.AppData
+	Errors  validation.List
+	Options actor.YesNoOptions
+}
+
+func CanEvidenceBeUploaded(tmpl template.Template) Handler {
+	return func(appData page.AppData, w http.ResponseWriter, r *http.Request, lpa *page.Lpa) error {
+		data := &canEvidenceBeUploadedData{
+			App:     appData,
+			Options: actor.YesNoValues,
+		}
+
+		if r.Method == http.MethodPost {
+			form := form.ReadHappyForm(r)
```

Comment:
Ah cool - I think I'll replace my custom yes/no form for this one to keep in line 👍

Reply:
it probably should be renamed to `YesNoForm` to make it obvious what it actually is